packages/browser-wallet/src/popup/pages/Account/AccountSettings/ExportPrivateKey.tsx

```diff
@@ -0,0 +1,92 @@
+import Button from '@popup/shared/Button';
+import CopyButton from '@popup/shared/CopyButton';
+import Form from '@popup/shared/Form';
+import FormPassword from '@popup/shared/Form/Password';
+import Submit from '@popup/shared/Form/Submit';
+import { TextArea } from '@popup/shared/Form/TextArea';
+import { usePrivateKey } from '@popup/shared/utils/account-helpers';
+import { selectedAccountAtom } from '@popup/store/account';
+import { sessionPasscodeAtom } from '@popup/store/settings';
+import { useAtomValue } from 'jotai';
+import React, { useState } from 'react';
+import { Validate } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import { absoluteRoutes } from '@popup/constants/routes';
+
+export default function ExportPrivateKey() {
+    const nav = useNavigate();
+    const { t: tSetup } = useTranslation('setup');
+    const { t: tPasscode } = useTranslation('changePasscode');
```

Comment:
This is fine for now, but maybe we need to restructure these translation.

Reply:
Yes, I think these shared translations could be done in a better way than this, but let's keep that for later.